cpp/include/cudf/detail/aggregation/aggregation.hpp

```diff
@@ -348,7 +355,10 @@ class std_var_aggregation : public aggregation {
  */
 class var_aggregation final : public std_var_aggregation {
  public:
-  var_aggregation(size_type ddof) : std_var_aggregation{aggregation::VARIANCE, ddof} {}
+  var_aggregation(size_type ddof)
+    : aggregation{aggregation::VARIANCE}, std_var_aggregation{aggregation::VARIANCE, ddof}
```

Comment:
You should only call constructor of the direct base class (`std_var_aggregation`). The class `std_var_aggregation` should be the one to call `groupby_aggregation{aggregation::VARIANCE}` which in turn calls `aggregation{aggregation::VARIANCE}`. The constructors should be called in chain.